indicators/discord-steam-0BFMGg.yml

```diff
@@ -0,0 +1,34 @@
+title: Discord/Steam Phishing Kit 0BFMGg
+description: |
+    Detects a phishing kit impersonating Discord and targeting Steam with a fake popup that opens when the "Get Nitro" button is clicked. The site promises to give you a free Discord Nitro subscription.
+    This phishing kit has been discovered by the FishFish.gg team.
+
+
+references:
+    - https://urlscan.io/result/a2bc1037-834a-4b09-8cba-81ad8850d053/
+    - https://urlscan.io/result/6e1af6f3-574d-465e-8ff9-2db5ba79f44a/
+    - https://urlscan.io/result/8cb54a81-49e5-430d-b504-283829a7fada/
+
+detection:
+
+    title:
```

Comment:
I think this rule can be improved to be more specific to the kit, the following background image from the in-line CSS seems like a better indicator than matching the generic `viewport` content


```
css|contains: 'assets/images/bg.svg?v=7583b449'
```